website/docs/indexing.md

```diff
@@ -27,27 +27,18 @@ could end up having to merge all the base files against all incoming updates/del
 Currently, Hudi supports the following index types. Default is SIMPLE on Spark engine, and INMEMORY on Flink and Java 
 engines.
 
-- **BLOOM:** Employs bloom filters built out of the record keys, optionally also pruning candidate files using 
-  record key ranges.Key uniqueness is enforced inside partitions.
-- **GLOBAL_BLOOM:** Employs bloom filters built out of the record keys, optionally also pruning candidate files using 
-  record key ranges. Key uniqueness is enforced across all partitions in the table.
-- **SIMPLE (default for Spark engines):** Default index type for spark engine. Performs a lean join of the incoming records against keys extracted from the table on 
-  storage. Key uniqueness is enforced inside partitions. 
-- **GLOBAL_SIMPLE:** Performs a lean join of the incoming records against keys extracted from the table on
-  storage. Key uniqueness is enforced across all partitions in the table.
-- **HBASE:** Manages the index mapping in an external Apache HBase table.
+- **BLOOM:** Uses bloom filters generated from record keys, with the option to further narrow down candidate files based on the ranges of the record keys. It requires keys to be partition-level or table/global-level unique so it can function correctly.
```

Comment:
"partition-level or table/global-level unique" this needs fix case by case, as in non-global index requires partition level uniqueness, and global index requires table-level uniqueness

Reply:
I understand- the indexes should be updated now